src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -0,0 +1,231 @@
+﻿namespace Nancy.Json.Simple
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+    using Nancy.Extensions;
+
+    /// <summary>
+    /// Nancy serialization stategy for SimpleJson
+    /// </summary>
+    public class NancySerializationStrategy : PocoJsonSerializerStrategy
+    {
+        private readonly bool retainCasing;
+        private readonly List<JavaScriptConverter> converters = new List<JavaScriptConverter>();
+        private readonly List<JavaScriptPrimitiveConverter> primitiveConverters = new List<JavaScriptPrimitiveConverter>();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NancySerializationStrategy"/> class.
+        /// </summary>
+        public NancySerializationStrategy() : this(false)

```

Comment:
I think there needs to be a `<remark>` comment on this, informing that using the default ctor means that `retainCasing` will be treated as `false`.